node-bb/README.md

```diff
@@ -0,0 +1,169 @@
+# TripleA Forums
+
+The software that powers the TripleA Forums is [NodeBB](https://github.com/NodeBB/NodeBB). 
+This document is instructions on how to build and run the Docker container of the forums using an 
+[official Docker container of NodeBB](https://hub.docker.com/r/nodebb/docker) as the base image.
+
+### Building
+
+1. Make a directory on your machine for where the config file and upload folder will live. ```mkdir -p /opt/triple-forums```
+1. Make a directory for uploaded files. Inside the upload directory, create folders category, emoji, files, profile, 
+sounds, and system.
+    ```
+    mkdir -p /opt/triple-forums/uploads/category
+    mkdir -p /opt/triple-forums/uploads/emoji
+    mkdir -p /opt/triple-forums/uploads/files
+    mkdir -p /opt/triple-forums/uploads/profile
+    mkdir -p /opt/triple-forums/uploads/sounds
+    mkdir -p /opt/triple-forums/uploads/system
+    ```
+1. Build the Docker image using the command ```docker build -t triplea-forums:latest .```
+
+### Running
+
+**First time run**
+
+You will need a running Mongo database locally or in a Docker container. See [MongoDB](#mongodb) for how to do this.
+
+1. Run the triplea-forums:latest image using the command 
+```docker run -d --rm --name triplea-forums -v /opt/triple-forums/uploads:/usr/src/app/public/uploads -p 8080:4567 triplea-forums:latest```
+1. Navigate to http://0.0.0.0:8080/. This is the install and configuration screen for NodeBB.
+1. Fill out the details on the page.
+1. Once the configuration is complete, you will need to save the config.json file off the container to the folder you 
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `4`, was `1`](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6342386)